.github/issue_template.md

```diff
@@ -0,0 +1,19 @@
+**Actual Behaviour**
```

Comment:
Maybe link to the contributing to scrapy document https://docs.scrapy.org/en/master/contributing.html? And maybe add a line that questions about how to use scrapy should be posted on Stackoverflow?